selenium-testing/config.properties.template

```diff
@@ -0,0 +1,7 @@
+portal_url=https://ofm-frontend-test-e1800b-dev.apps.silver.devops.gov.bc.ca/login/
```

Comment:
At some point, you might want to add a quick README.md explaining how to do things like setup the local config.properties and run tests.

Reply:
Done: https://github.com/bcgov/ecc-ofm/pull/319/commits/bce2de80d5de06d049add83578bcdae3a627059e